internal/app/singularity/remote_get_login_password.go

```diff
@@ -0,0 +1,53 @@
+package singularity
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	scslibclient "github.com/sylabs/scs-library-client/client"
+)
+
+// RemoteGetLoginPassword retrieves cli token from oci library shim
+func RemoteGetLoginPassword(config *scslibclient.Config) (string, error) {
+	client := http.DefaultClient
+	path := "/v1/rbac/users/current"
+	endPoint := config.BaseURL + path
+
+	req, err := http.NewRequest(http.MethodGet, endPoint, nil)
+	if err != nil {
+		return "", fmt.Errorf("error creating request: %v", err)
+	}
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", config.AuthToken))
+	res, err := client.Do(req)
+	if err != nil {
+		return "", fmt.Errorf("client encountered error making request to server: %v", err)
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
+		return "", fmt.Errorf("unexpected response from server, status: %v", res.Status)
+	}
```

Comment:
For backwards compat, we should probably change the error message when a 404 comes out from the server and say the same as `This library does not support direct OCI access, status: %v`